### PR TITLE
Story3task1

### PR DESCRIPTION
**Pull request by LottieRugg**
Hello, this code is to show this list of menu items. 
I haven't done the media query for desktop yet tho

### CODE REVIEW

File: src/components/Item_card/Item_card.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+.item_calories{
+  background-color: #17a2b8;
+  @include Item_details;
+  color: #ffffff;
+}
+
+.item_type{
+  background-color:#007bff;
+ @include Item_details;
```
Comment (Mead-y) [inline src/components/Item_card/Item_card.scss:31]: Indent by 1 to line up with the others

File: src/components/Item_card/Item_card.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  color: #ffffff;
+}
+
+.item_category{
+  background-color: #ffc107;
+ @include Item_details
```
Comment (Mead-y) [inline src/components/Item_card/Item_card.scss:37]: Indent by 1 again so it lines up :)

File: src/components/Item_card/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import './Item_card.css'
+
+const Item_card = (foodItem) => {
+    return(
+        <>
+            <div className = 'Item_card'>
```
Comment (Mead-y) [inline src/components/Item_card/index.js:6]: Consider changing Item_card to item_card to be consistent with the other names

File: src/components/Item_card/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+                    {foodItem.side ?
+                    <p className = 'item_category'>
+                        side
+                    </p> :''}
+                </div>
+                <div className= 'price'>
+                    £{foodItem.price}
+                </div>
+
+
```
Comment (Mead-y) [inline src/components/Item_card/index.js:26]: thin out the few lines of empty code

File: src/components/Item_card/index.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            </div>
+        </>
+
+    )}
```
Comment (Mead-y) [inline src/components/Item_card/index.js:30]: Put the } on a new line and get rid of the empty line (line 29) (This is stuff Ash picked us up for so just trying to spot stuff he might pick us up on)

File: src/components/Item_card/index.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
```
Comment (Mead-y) [inline src/components/Item_card/index.js:31]: Keep the empty line above export default though (sorry if this is confusing)

File: src/components/Order_page/index.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import {useEffect, useState} from "react";
```
Comment (Mead-y) [inline src/components/Order_page/index.js:1]: Get rid of all semi colons to keep with consistency of the other files (we went through and deleted most of hem earlier)

File: src/components/Order_page/index.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    const [foodItemData, setItemData] = useState([])
+
+    useEffect( () =>{
+
+    fetch('http://localhost:8080/restaurants/1')
+        .then(res => res.json())
+        .then(function (data) {
+            setItemData(data.foodItems)
+        })
+
```
Comment (Mead-y) [inline src/components/Order_page/index.js:13]: Empty line

File: src/components/Order_page/index.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+}, [])
+
+    let foodItem = foodItemData.map((foodItem) => {
+        return <Item_card price={foodItem.price} calories={foodItem.calories}
+                          foodName={foodItem.foodName} foodType ={foodItem.foodType}
```
Comment (Mead-y) [inline src/components/Order_page/index.js:18]: Indentation here

File: src/components/Order_page/index.js [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+        side ={foodItem.sideItem}/>
+    })
+    return (
+        <>console.log(foodItem)
```
Comment (Mead-y) [inline src/components/Order_page/index.js:22]: Here is the mysterious console.log that was displaying on the page

**Review by Mead-y — CHANGES_REQUESTED** [review]
Few changes, give us a shout if you need anything explaining.

File: src/components/Item_card/index.js (hunk 6 above)
Comment (LottieRugg) [reply src/components/Item_card/index.js:31]: 😅 think i got u

**Pull request merged**